.github/workflows/smoke.yaml

```diff
@@ -2,7 +2,7 @@ name: Smoke test
 
 on:
   push:
-    branches: ['**']
+    branches: ['**', '!debug/**']
```

Comment:
why exclude tests in debug?

Reply:
This is meant to be used to speed up developing workflow, not meant to be a final change, see WIP